packages/metadata-common/src/MetadataEditor.tsx

```diff
@@ -110,6 +114,16 @@ export class MetadataEditor extends ReactWidget {
     this.update();
   }
 
+  private isValueEmpty(schemaValue: any): boolean {
+    return (
+      schemaValue === undefined ||
+      schemaValue === null ||
+      schemaValue === '' ||
```

Comment:
We can just use `!schemaValue` instead of these 3 value checks

Reply:
I mentioned this in a review comment from Luciano but we can't use that because it would evaluate 0 and false as falsy as well, but those might be valid content.